### DISCUSSION TITLE
Multiline support when parsing an array to yaml

### DISCUSSION THREAD

**Discussion by adrianocastro189** [dallgoot/yaml#14] [Q&A]
Hey there!

I'm trying to convert an array to yaml in a test method with a very simple structure like this:

```php
$value = ['test-key-1' => "this is a test with\r\na line break"]
```

When I do `Yaml::dump($value)`, the output is:

```yaml
test-key-1: this is a test with\r\na line break
```

Question: shouldn't it handle the line break as:

```yaml
test-key-1: |
  this is a test with
  a line break
```

I may be missing something when calling the `dump` method, perhaps an option. But I tried a few things and can't achieve that. Could anyone shine some light on this? Thanks!

**Comment by dallgoot** [comment]
Hello, sorry for the late response.
The string as you declared it is valid in itself : you can specify strings in multiple ways.
The fact that you're seeing the line breaks as-is is strange though : were you using var_dump to see the value ?

**Reply by adrianocastro189** [reply]
@dallgoot no worries at all and I appreciate your answer.
At that time I posted this question, I installed `symfony/yaml` on my application and it covered the cases I needed, so it's all good and the application is working fine.
